instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Comment  = props => {
+    return(
+        <div>
+            <span>{props.username} </span>
+            <span>{props.text}</span>
+        </div>
+    )
+}
+
+Comment.propTypes= {
+    username: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
```

Comment:
great use of `isRequired` here to make sure an alert is thrown to the user when incorrect instead of just a console message